README.markdown

```diff
@@ -296,15 +274,6 @@ func reticulateSplines(spline: [Double]) -> Bool {
 }
 ```
 
-For functions with long signatures, add line breaks at appropriate points and add an extra indent on subsequent lines:
```

Comment:
My reasoning for everyone:

Xcode's "page guide" is an illusion, the display's and/or window's width is the true limit. Because this width is never constant, a text editor's way of breaking lines is the one true form of line breaking.

And I also tend to agree with this (we are trained to read a bunch of words in a single line, not in columns): https://twitter.com/lllucas/status/592905829583556611

![](https://pbs.twimg.com/media/CDpstcXWYAANkSP.png:large)